ext/blake2_ext/blake2-impl.h

```diff
@@ -1,20 +1,23 @@
 /*
```

Comment:
Well, the rake task would be OK as long as it only copied the files needed to the correct place.  I would recommend keeping them vendored in your gem though since that allows you to verify that you have the correct files in place and that they have not been modified.

If you just pull them in blindly when you build you risk a MITM on those files being modified (or not available any longer, like when they recently re-hosted them to Github).